src/DetailView.js

```diff
@@ -0,0 +1,16 @@
+import { VivViewerLayer } from './layers';
+import VivView from './VivView';
+
+/**
+ * This class generates a VivViewerlayer and a view for use in the VivViewer as a detailed view
+ * */
+export default class DetailView extends VivView {
+  getLayer({ props }) {
+    const { loader } = props;
+    const { id } = this;
+    return new VivViewerLayer(props, {
+      id: `${loader.type}-${id}#`,
```

Comment:
If one id contains another (like `detail` and `detail2`, we need a way to mark the end of the string.  This was my solution since there are only 2 parts of the `id` at this level.

Reply:
Cool: That scheme seems fine, I think it just needs to be wrapped so the details aren't exposed.